licenses/git.go

```diff
@@ -46,7 +46,7 @@ type GitRepo struct {
 // FindGitRepo finds the Git repository that contains the specified filePath
 // by searching upwards through the directory tree for a ".git" directory.
 func FindGitRepo(filePath string) (*GitRepo, error) {
-	path, err := findUpwards(filepath.Dir(filePath), gitRegexp, srcDirRegexps, nil)
+	path, err := findUpwards(filepath.Dir(filePath), gitRegexp, "/", nil)
```

Comment:
`/` might be too broad. IIUC, I think we can limit this to the current directory and the mod cache folder?

Reply:
Will do